LOGGING/src/EventSourceListener/EventSourceListener.csproj

```diff
@@ -26,7 +26,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\..\BASE\src\Microsoft.ApplicationInsights\Microsoft.ApplicationInsights.csproj" />
-    <PackageReference Include="System.Diagnostics.Process" Version="4.1.0" />
+    <PackageReference Include="System.Diagnostics.Process" Version="4.3.0" />
```

Comment:
Do we really use this?

Reply:
i just checked, looks like this isn't used. i'll submit a PR to remove it.